<?xml version="1.0"?>
<launch>
  <arg name="package_name" default="teaching_processing"/>
  <arg name="pcd_name" default="test"/>
  <arg name="pc_src" default="/photoneo_center/pointcloud"/>

  <node pkg="teaching_processing" type="save_pcd" name="save_pcd_node">
    <param name="package_name" value="$(arg package_name)"/>
    <param name="pcd_name" value="$(arg pcd_name)"/>
    <param name="pc_src" value="$(arg pc_src)"/>
  </node>
</launch>